### ISSUE TITLE
statsmodels.distributions.edgeworth.ExpandedNormal - 4.cumulant higher than 4

### ISSUE THREAD

**Issue by cube2022** [statsmodels/statsmodels#8326]
In line 166 of edgeworth.py  is checked if the imag-part is zero and if abs(r) is smaller than 4. If I choose a fourth cumulant higher than 4 I get this warning and yes in this case the imag-parts are zero and abs(r) is smaller 4. But: where does this limit of 4 comes from? 
Is it based on a paper? 
By the way: I assume that the first cumulant is zero and the second is 1. (scaling, centering in line 163,164) I use statsmodels 0.13.2

**Comment by josef-pkt** [comment]
original PR by @ev-br is #1325

I haven't looked at this in a long time, and don't remember details.
Maybe Evgeni found the condition and bound in one of the cited articles.

The problem is that the simple orthogonal polynomials can have negative pdf intervals and nonmonotonic cdf if the distribution is too far away from the base distributions.
I had looked at several methods to "fix" those negative regions, but they all require numerical integration to compute or adjust the integration constant, AFAIR. This removes the main advantage of orthogonal polynomials which are simple to compute.


> By the way: I assume that the first cumulant is zero and the second is 1. (scaling, centering in line 163,164)

The first to cumulants are mean and variance
line 190 :      `mu, sigma = cum[0], np.sqrt(cum[1])`

but the polynomials are computed in terms of the standardized distribution, loc=0, scale=1.

**Comment by cube2022** [comment]
> You can check by computing the pdf on a grid in the possibly affected region.
You mean to vary the value of the cumulants and check the different methods, similiar to a Monte Carlo?

**Comment by ev-br** [comment]
I don't remember details, sorry. The git history of my sandbox repo (github remembers!) shows a separate commit, but not much else. It's possible that the threshold isfrom some playing around for a couple of distrubtions. 

https://github.com/ev-br/edgeworth/commits/master

**Comment by josef-pkt** [comment]
@cube2022 
no, compute the pdf for different values of x for given cumulants with a 4th cumulant like 5.
i.e. compute and plot the pdf(x) function for a parameter that might have a negative valued region.
